src/Compilers/CSharp/Portable/Lowering/LocalRewriter/LocalRewriter_Call.cs

```diff
@@ -1113,18 +1116,76 @@ bool isLambdaConversion(BoundExpression expr)
             { WasCompilerGenerated = true };
         }
 
-        /// <summary>
-        /// To create literal expression for IOperation, set localRewriter to null.
-        /// </summary>
-        private static BoundExpression MakeLiteral(SyntaxNode syntax, ConstantValue constantValue, TypeSymbol type, LocalRewriter? localRewriter)
+        private static BoundExpression CreateParamsSpan(
+            SyntaxNode syntax,
+            TypeSymbol paramArrayType,
+            ImmutableArray<BoundExpression> arrayArgs,
+            CSharpCompilation compilation,
+            BindingDiagnosticBag diagnostics)
         {
-            if (localRewriter != null)
+            var elementType = paramArrayType.GetParamsElementType().Type;
+
+            // PROTOTYPE: If we actually allocate from the stack, it may be necessary to
+            // clear the execution stack before allocation. (See corresponding requirement
+            // for 'stackalloc' in LocalRewriter.VisitStackAllocArrayCreationBase().)
+
+            // For now, simply allocate the array on the heap: new Span<T>(new T[length] { ... })
+            var array = CreateParamArrayArgument(
+                syntax,
+                ArrayTypeSymbol.CreateSZArray(compilation.SourceAssembly, TypeWithAnnotations.Create(elementType)),
+                arrayArgs,
+                compilation,
+                diagnostics);
+
+            var spanConstructor = (MethodSymbol)Binder.GetWellKnownTypeMember(compilation, WellKnownMember.System_Span_T__ctorArray, diagnostics, syntax.Location);
+            if (spanConstructor is null)
             {
-                return localRewriter.MakeLiteral(syntax, constantValue, type);
+                return createBadExpression(syntax, paramArrayType, arrayArgs);
             }
-            else
+
+            var spanType = spanConstructor.ContainingType.Construct(elementType);
+            spanConstructor = spanConstructor.AsMember(spanType);
+
+            BoundExpression span = new BoundObjectCreationExpression(
+                syntax,
+                spanConstructor,
+                ImmutableArray.Create(array))
+            { WasCompilerGenerated = true };
+
+            if (paramArrayType.IsReadOnlySpanType(compilation))
+            {
+                var spanToReadOnlySpanOperator = (MethodSymbol)Binder.GetWellKnownTypeMember(compilation, WellKnownMember.System_Span_T__op_Implicit_SpanReadOnlySpan, diagnostics, syntax.Location);
```

Comment:
Consider adding test with missing newly-added well-known members System_Span_T__op_Implicit_SpanReadOnlySpan and System_Span_T__ctorArray